sdks/java/io/hadoop-format/src/test/java/org/apache/beam/sdk/io/hadoop/format/HadoopFormatIOSequenceFileTest.java

```diff
@@ -0,0 +1,312 @@
+/*
```

Comment:
@b923 Could you get back HadoopFormatIOIT? This is an integration test, which runs on real environment and written in similar way as for other IOs. It allows to write generated data and check if data was not lost or corrupted by comparing a hash value computed over all dataset. We can extend it further to test different OutputFormat's.

Reply:
`HadoopFormatIOIT` returned back